src/main/java/nl/novi/techiteasy1121/exceptions/UsernameNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package nl.novi.techiteasy1121.exceptions;
```

Comment:
Zorg ervoor dat je exception handling consistent is in alle controllers. Het toevoegen van specifieke exception classes zoals UsernameNotFoundException is een goede praktijk.